week1_gp6_zad2.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+using namespace std;
+unsigned int SIZE=100;
+
+struct OrderedPair{
+    int first;
+    int second;
+};
+
+void PrintOrderedPair(const OrderedPair& PairToPrint){
+    cout<<"("<<PairToPrint.first<<"; "<<PairToPrint.second<<")"<<endl;
+}
+
+bool isFirstBigger(const OrderedPair& FirstPair, const OrderedPair& SecondPair){
+    if(FirstPair.first>SecondPair.first || (FirstPair.first==SecondPair.first && FirstPair.second>SecondPair.second))
```

Comment:
You don't really need the `if` statement here. You can replace this one directly with a `return` statement :))